app/static/js/slcNewProposal.js

```diff
@@ -86,17 +86,42 @@ function fixStepIndicator(navigateTab) {
 
 // TODO: empty the courseInstructor input after an instructor has been added to the table.
 function callback() {
-  let instructor = $("#courseInstructor").val();
+  let data = $("#courseInstructor").val();
+  data = data.split(",");

```

Comment:
Rather than reassign data from text to an array, I think the assignment line should do the split().

Reply:
Hila had made this function to validate the proposal form however she mentioned that it would be more effective if we created a more general validation and I assumed that would fall under a separate issue.

Reply:
Fixed in next push